vim/merlin/autoload/merlin.py

```diff
@@ -246,6 +246,11 @@ def differs_from_current_file(path):
     buf_path = vim.eval("expand('%:p')")
     return buf_path != path
 
+def fnameescape(s):
+    # this will not work for filenames with quotes
+    # but to fix that need fnameescape defined on python side
+    # at least it will handle spaces
+    return vim.eval("fnameescape('%s')" % s)
```

Comment:
I think you can probably do some form of escaping on `s` from python, we already do this in some places (search for `.replace`).